### PR TITLE
spm update_repo doesn't always require arguments

### PR DESCRIPTION
**Pull request by techhat**

### CODE REVIEW

**Comment by techhat** [comment]
Ping @gravyboat.

File: salt/utils/parsers.py [hunk 1]
```diff
@@ -2637,8 +2637,9 @@ def _mixin_setup(self):
     def _mixin_after_parsed(self):
         # spm needs arguments
         if len(self.args) <= 1:
-            self.print_help()
-            self.exit(salt.defaults.exitcodes.EX_USAGE)
+            if self.args[0] not in ('update_repo'):
```
Comment (s0undt3ch) [inline salt/utils/parsers.py]: You're missing a comma to make it iterable, `('update_repo',)`

**Comment by techhat** [comment]
Added, though it did iterate fine for me in my tests.

**Comment by basepi** [comment]
This was inadvertently closed. Sorry!

**Comment by s0undt3ch** [comment]
Go Go Jenkins!

**Comment by gravyboat** [comment]
@techhat Unless you have objections I believe we can close https://github.com/saltstack/salt/issues/25475 now.

**Pull request merged**